Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```
@@ -402,31 +419,27 @@ script:
       name: reputation
     - description: Alert type. Supports comma-separated values.
       name: alert_type
-    - description: Alert category. Supports comma-separated values.
-      name: alert_category
-    - description: Alert workflow. Supports comma-separated values.
-      name: workflow
-    - description: Device name. Supports comma-separated values.
+    - description: Device name. Supports comma-separated values. Note, the slashes in the device name should be doubled.
       name: device_name
-    - description: Process name. Supports comma-separated values.
+    - description: Process name. Supports comma-separated values. Note, the slashes in the process name should be doubled.
       name: process_name
-    - auto: PREDEFINED
-      defaultValue: first_event_time
-      description: Field by which to sort the results. Can be "first_event_time", or "last_event_time".
+    - description: Field by which to sort the results. Can be "first_event_timestamp", or "last_event_timestamp".
```

```suggestion
    - description: Field by which to sort the results. Either "first_event_timestamp" or "last_event_timestamp".
```

We can also skiip the second sentance as the options are limited anyway. Your call.